Formula/click.rb

```diff
@@ -0,0 +1,18 @@
+class Click < Formula
+  desc "The command-line interactive controller for Kubernetes"
+  homepage "https://github.com/databricks/click"
+  url "https://github.com/databricks/click/archive/v0.3.2.tar.gz"
+  sha256 "eed648409bf78a05658a9d097e5099ca17bf19df70122e2067859ae94c5575d5"
+  head "https://github.com/databricks/click.git"
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/click"
+  end
+
+  test do
+    # A better test would requrie a full K8s cluster, unfortunately
+    system bin/"click", "-V"
```

Comment:
Can we check for an error instead?

Reply:
Still working on this -- need to figure out how to use expect.

Reply:
I think it's more dangerous to try to check for a non-existent cluster due to too many network dependencies.  Instead, I setup fake configuration files and at least confirm that it starts in the correct, default, state.  Is that good enough?